test/dynamo/test_higher_order_ops.py

```diff
@@ -2501,6 +2502,75 @@ def fn(pred, pytree_in):
             ):
                 torch.compile(fn, backend="eager")(pred, pytree_in)
 
+    def test_hints_wrapper(self):
```

Comment:
Can we add more tests for pytree inputs? I think current diff might not be enough to cover pytree inputs. Can check _higher_order_ops/map.py:map_wrapper for some inpiration